Makefile

```diff
@@ -294,12 +294,24 @@ lint-license: pull-buildenv buildenv-dirs
 "$(GOBIN)/addlicense":
 	go install github.com/google/addlicense@v1.0.0
 
-"$(GOBIN)/kustomize":
-	CGO_ENABLED=0 go install sigs.k8s.io/kustomize/kustomize/v5@$(KUSTOMIZE_VERSION)
-
-# install kustomize binary for containerized testing
-"$(BIN_DIR)/kustomize": "$(GOBIN)/kustomize"
-	cp $(GOBIN)/kustomize $(BIN_DIR)/kustomize
+.PHONY: clean-kustomize
+clean-kustomize:
+	@ rm -rf $(KUSTOMIZE_STAGING_DIR)
+	@ rm -rf $(BIN_DIR)/kustomize
+
+KUSTOMIZE_URL := gs://config-management-release/config-sync/kustomize/tag/$(KUSTOMIZE_VERSION)/kustomize-$(KUSTOMIZE_VERSION)-linux-amd64.tar.gz
+KUSTOMIZE_STAGING_DIR := $(OUTPUT_DIR)/third_party/kustomize
+KUSTOMIZE_TARBALL := /tmp/kustomize-$(KUSTOMIZE_VERSION)-linux-amd64.tar.gz
+
+"$(BIN_DIR)/kustomize": clean-kustomize buildenv-dirs
+	@ gsutil cp $(KUSTOMIZE_URL).sha256 $(KUSTOMIZE_TARBALL).sha256
```

Comment:
nit: for consistency, can you remove the space between @ and the command?

Reply:
done